File: 100-Days-Series/Day 65/DiamandPatternWithNumber.java

public class DiamandPatternWithNumber {
    // Function to print Diamand Pattern With Numbers.
    public static void getPrintPattern(int rows){
        // Upper Part
        for(int i=1;i<=rows;i++){
            for(int space=1;space<=rows-i;space++){
                System.out.print(" ");
            }
            for(int j=1;j<=(2*i-1);j++){
                System.out.print(i);
            }
            System.out.println();
        }
        // Lower Part 
        for(int i=rows-1;i>=1;i--){
            for(int space=1;space<=rows-i;space++){
                System.out.print(" ");
            }
            for(int j=1;j<=(2*i-1);j++){
                System.out.print(i);
            }
            System.out.println();
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start Here
        int rows;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        rows=sc.nextInt();
        getPrintPattern(rows);
    }
}

File: 100-Days-Series/Day 65/DiamandPatternWithNumber.js
// Arrow Function to print Diamand Pattern With Numbers.
const getPrintPattern=(rows)=>{
    // Upper Part 
    for(let i=1;i<=rows;i++){
        line="";
        for(let space=1;space<=rows-i;space++){
            line+=" ";
        }
        for(let j=1;j<=(2*i-1);j++){
            line+=i;
        }
        console.log(line);
    }
    // Lower Part
    for(let i=rows-1;i>=1;i--){
        line="";
        for(let space=1;space<=rows-i;space++){
            line+=" ";
        }
        for(let j=1;j<=(2*i-1);j++){
            line+=i;
        }
        console.log(line);
    }
}

// Let's Start Here
let rows=5;
getPrintPattern(rows);